src/js/components/Anchor/__tests__/Anchor-test.js

```diff
@@ -153,6 +153,7 @@ describe('Anchor', () => {
   test('size renders', () => {
     const component = renderer.create(
       <Grommet>
+        <Anchor size="10px" />
```

Comment:
Can you add this new Anchor line at the end of the list of anchors so it won't impact all the snapshots of the following anchors?

Reply:
yes! :)